### ISSUE TITLE
Add descriptive analysis content

### ISSUE THREAD

**Issue by mbcann01** [brad-cannell/course-Introduction-to-R-Programming-for-Epidemiologic-Research#50]
Start by copying and pasting content from the SAS course into R markdown files that will render in HTML.

We will iteratively improve the content over time.

I think for now we won't include either of the "graphical description" lessons. They already get graphical descriptions in the data visualization learning module. That means that for now, the descriptive analysis learning module will include:

- Introduction to Descriptive Analysis
- Describing Categorical Variables
- Describing Central Tendency
- Describing Dispersion
- Describing the Relationship Between a Continuous Outcome and a Continuous Predictor
- Describing the Relationship Between a Continuous Outcome and a Categorical Predictor
- Describing the Relationship Between a Categorical Outcome and a Categorical Predictor

- [x] Copy over Descriptive Analysis I Module Overview
- [x] Copy over Numerical Descriptions of Categorical Variables
- [x] Copy over Graphical Descriptions of Categorical Variables
- [x] Copy over Measures of Central Tendency
- [x] Copy over Measures of Dispersion
- [x] Copy over Graphical Descriptions of Numerical Variables
- [x] Copy over Continuous Outcome and Continuous Predictor
- [x] Copy over Continuous Outcome and Categorical Predictor
- [x] Copy over Categorical Outcome
- [x] Create links to individul lesson markdown files on the module overview page - use new names above
- [x] Edit Descriptive Analysis I Module Overview
- [x] Edit Numerical Descriptions of Categorical Variables
- [x] Edit Graphical Descriptions of Categorical Variables
- [x] Edit Measures of Central Tendency
- [x] Edt Measures of Dispersion
- [x] Edit Continuous Outcome and Continuous Predictor
- [x] Edit Continuous Outcome and Categorical Predictor
- [x] Edit Categorical Outcome
- [x] Add message=FALSE to all the Tidyverse code chunks

**Comment by mbcann01** [comment]
Should I add R Fiddle chunks to the HTML documents?

**Comment by mbcann01** [comment]
Left off at cleaning up numerical_descriptions_of_categorical_variables

**Comment by mbcann01** [comment]
Left off at trying to recreate the video section in graphical descriptions of categorical variables.

**Comment by mbcann01** [comment]
When I left off, I was making graphics for the measures of central tendency notebook
* Open the PowerPoint file in the img folder
* Manually (using positiioning panel) make sure all the boxes stay aligned when you delete the traits of the mean, median, and mode.
* Export as png.
* Embed into notebook

**Comment by mbcann01** [comment]
Left off explaining how the mode function works.

**Comment by mbcann01** [comment]
Stuff I cut from measures of central tendency because R and SAS work differently and this isn't strictly about central tendency.  However, I think I need to show them how to calculate all of these stats together as would happen in SAS. I might need to create a new lesson.

![image](https://app.gitkraken.com/api/glo/boards/5c5b17363cc4410015e926a9/attachments/5d2cf40013853d0011aa98dd)
Stuff I cut from measures of central tendency because R and SAS work differently and this isn't strictly about central tendency.  However, I think I need to show them how to calculate all of these stats together as would happen in SAS. I might need to create a new lesson.

Here is the summary report created by SAS. As you can see, by default SAS calculates the N, the mean, the standard deviation, the minimum value, and the maximum value for every numerical variable in the data set. The first column gives the variable names. Here, N is the number of observations with a non-missing value for this particular variable. 
Before we start refining our SAS code, I want to point out the utility of using PROC MEANS to check your data. And what do I mean by that? Well, we’ve already talked a little bit about data checking. Often times, for various reasons, some observations for a given variable take on values that don’t make sense. PROC MEANS is one tool you can use to try to figure out if you have values in your data that don’t make sense. 



For example, as you scan this report, do you see any red flags? Look at the weight variable. Do you really think a mean weight of 1,113 pounds sounds reasonable? This should definitely be a red flag for you. Now move your gaze three columns to the right and notice that the maximum value of weight is 19,000 lbs – an impossible value for a study in human populations. In this case the real weight was supposed to be 190 pounds, but the person entering the data accidently got a little trigger-happy with the zero key. Now in this case we can just go back and fix our data, but what if we didn’t know this value was an error? What if it were a value that was technically possible, but very unlikely? Well, we can’t just go changing values in our data. It’s unethical, and in some cases illegal.

**Comment by mbcann01** [comment]
Should I show them how to do descriptives for many variables at once using purrr?

**Comment by mbcann01** [comment]
dispersion_01_people

**Comment by mbcann01** [comment]
Left off on line 185

**Comment by mbcann01** [comment]
Create normal plot

**Comment by mbcann01** [comment]
Left off around 163 -- explain the output.

**Comment by mbcann01** [comment]
* Add slides to cont and cont
* Proofread cont and cont

**Comment by mbcann01** [comment]
Add categorized age group and bmi directly to the class data in 05_cont_cont

**Comment by mbcann01** [comment]
- [ ] Proofread Continuous Outcome and Categorical Predictor
I was tired when I finished it.

**Comment by mbcann01** [comment]
- [ ] Come up with better example than systolic and hypertension
